.github/workflows/build.yml

```diff
@@ -390,13 +518,12 @@ jobs:
         fi
 
     - name: Release
-      if: github.ref_type == 'tag'
```

Comment:
Why is this removed?

Reply:
Because the step was moved to another job (`publish-release`), that already checks for running on a tag.